chainer/variable.py

```diff
@@ -523,6 +536,12 @@ def __init__(self, data=None, **kwargs):
                     array_types[-1], type(data))
                 raise TypeError(msg)
 
+        self._init_attrs(data, name, grad, requires_grad, None)
+
+    def _init_attrs(self,
```

Comment:
The name of this method also has a room to discuss.
I think "attributes" is nonessential. How about `_init_common`?

Reply:
Hmm, what about `_init_impl`?

Reply:
That also sounds good to me.